pretests/log_versions/log_versions.py

```diff
@@ -0,0 +1,50 @@
+r"""
+Summary
+-------
+
+Preserve docker versions in state files.
+
+Operational Summary
+-------------------
+
+#. Run 'docker version'; use DockerVersion module to parse output
+#. Run 'rpm -q docker|docker-latest' (whichever one is in use)
```

Comment:
Since this is not relevant to my current purpose, I would prefer to deal with this in a separate PR.